arkanoid.c

```diff
@@ -12,6 +12,7 @@
 
 Ball ball;
 VAUS vaus;
+uint8_t level_number;
```

Comment:
This would be better in levels.c script
levels.c should manage everything revolving around levels

Reply:
RIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIGHT 😎